src/main/target/STM32_UNIFIED/target.h

```diff
@@ -316,6 +346,14 @@
 #define USE_RX_SPEKTRUM
 #define USE_RX_SPEKTRUM_TELEMETRY
 
+#define USE_RX_SPI
+#define USE_RX_EXPRESSLRS
+#define USE_RX_SX1280
+#if (TARGET_FLASH_SIZE > 512) || defined(STM32F7X2_SPIRX) || defined(STM32G47X_SPIRX)
```

Comment:
so we won't have SX127x support on F411 target?

Reply:
Found a way to make the SX127x also fit by adjusting the STM32F411 loader code. This will give us some more space in first sector of the flash. Custom defaults don't use much of the actual size and there is still some more space.